diff --git a/src/cpp/desktop/3rdparty/qtsingleapplication/qtlocalpeer.cpp b/src/cpp/desktop/3rdparty/qtsingleapplication/qtlocalpeer.cpp
index 09ab680..97774eb 100644
--- a/src/cpp/desktop/3rdparty/qtsingleapplication/qtlocalpeer.cpp
+++ b/src/cpp/desktop/3rdparty/qtsingleapplication/qtlocalpeer.cpp
@@ -41,6 +41,7 @@
 
 #include "qtlocalpeer.h"
 #include <QCoreApplication>
+#include <QDataStream>    // Patch for building against Qt 5.5.1
 #include <QTime>
 
 #if defined(Q_OS_WIN)
