--- src/cpp/CMakeLists.txt.orig	2016-02-02 16:46:10.330248089 -0700
+++ src/cpp/CMakeLists.txt	2016-02-02 16:47:36.962809778 -0700
@@ -80,7 +80,7 @@
       add_definitions(-D_FORTIFY_SOURCE=2)
       add_definitions(-fstack-protector --param ssp-buffer-size=4)
       add_definitions(-pie -fPIE)
-      set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,-z,relro,-z,now")
+      set(CMAKE_EXE_LINKER_FLAGS "-Wl,-z,relro,-z,now ${CMAKE_EXE_LINKER_FLAGS}")
    endif()
 
 # Win32 specific global directives
