--- src/cpp/desktop/CMakeLists.txt.orig	2016-02-02 16:49:08.551993985 -0700
+++ src/cpp/desktop/CMakeLists.txt	2016-02-02 16:49:50.554953192 -0700
@@ -66,8 +66,7 @@
 endif()
 
 # set the cmake prefix path based on the location of the qmake executable
-get_filename_component(QT_BIN_DIR ${QT_QMAKE_EXECUTABLE} PATH)
-set(CMAKE_PREFIX_PATH "${QT_BIN_DIR}//..//lib//cmake")
+set(CMAKE_PREFIX_PATH "/usr/lib/cmake")
 
 # find and include Qt
 add_definitions(-DQT_NO_CAST_FROM_ASCII -DQT_NO_CAST_TO_ASCII)
